app/controllers/users_controller.rb

```diff
@@ -127,7 +127,13 @@ def validate_attributes
   end
 
   def valid_phone_number?(number)
-    number.to_i.to_s == number && number.length >= 11
+    numeric?(number) && number.length >= 11
+  end
+
+  def numeric?(str)
```

Comment:
I think we use this method somewhere else to check for numbers and from what I understand regex can be quite slow for failing strings and I find this to be a bit more readable. Let me know if you feel strongly about this tho

Reply:
I was looking at this, but I think you're right. I thought it would be way longer. Thanks! 
https://mentalized.net/journal/2011/04/14/ruby-how-to-check-if-a-string-is-numeric/